micropip/_compat_not_in_pyodide.py

```diff
@@ -64,45 +63,6 @@ def canonicalize_name(name: str) -> str:
     return _canonicalize_regex.sub("-", name).lower()
 
 
-# Vendored from pip
-class UnsupportedWheel(Exception):
-    """Unsupported wheel."""
-
-
-def wheel_dist_info_dir(source: ZipFile, name: str) -> str:
```

Comment:
I think there is not much reason to import this from pyodide. I move it to `metadata.py`.